Core/System.cpp

```diff
@@ -767,6 +767,9 @@ void InitSysDirectories() {
 	File::CreateDir(GetSysDirectory(DIRECTORY_SAVEDATA));
 	File::CreateDir(GetSysDirectory(DIRECTORY_SAVESTATE));
 	File::CreateDir(GetSysDirectory(DIRECTORY_SYSTEM));
+#if PPSSPP_PLATFORM(UWP) && !defined(__LIBRETRO__)
```

Comment:
Some Xbox users asked for this actually, also for `Cheats` folder for easy access, to avoid creating it manually.
if it's unnecessary we can drop this, I don't see it that important, but since I don't use Xbox don't know if it's really needed.

Reply:
Well, I'm fine with it, I just think there's no reason for it to be different for UWP.

I do think cheats should be easier to import...

-[Unknown]